### PR TITLE
Regenerate with latest gir

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Needed by https://github.com/gtk-rs/gtk3-rs/pull/798.

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Updated with new gir fix and added missing manual types.

File: gtk/src/image.rs [hunk 1]
```diff
@@ -126,134 +125,134 @@ impl ImageBuilder {
     /// Build the [`Image`].
     #[must_use = "Building the object from the builder is usually expensive and is not expected to have side effects"]
     pub fn build(self) -> Image {
-        let mut properties: Vec<(&str, &dyn ToValue)> = vec![];
+        let mut builder = glib::Object::builder::<Image>();
```
Comment (sdroege) [inline gtk/src/image.rs:128]: This should be a plain wrapper around the `ObjectBuilder` like the autogenerated builders now. Same for the other manual builders

**Comment by sdroege** [comment]
Looks good otherwise

File: gtk/src/image.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply gtk/src/image.rs]: Oh nice!

**Comment by GuillaumeGomez** [comment]
Updated builders.

**Review by sdroege — APPROVED** [review]

**Pull request merged**